canonical/vanilla-framework
# Adjust columns on mobile view for step-list--detailed

## Done

- Sets columns to 6 at a smaller breakpoint

Fixes https://github.com/canonical/vanilla-framework/issues/5002

## QA

- Open [demo](https://vanilla-framework-5007.demos.haus/docs/examples/patterns/lists/lists-stepped-detailed)
- Check that there is the content and title of the stepped list take the appropriate number of columns
- On merge remove this from ubuntu.com https://github.com/canonical/ubuntu.com/pull/13580/files#diff-f5a5b0429dba7d86dd96396c39539ab3a091d90536bccb9db008a3dbc487f83dR1506

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

Mobile
![image](https://github.com/canonical/vanilla-framework/assets/58276363/ae3c18f6-71bd-4388-acc3-2c5073fa01f1)

Tablet
![image](https://github.com/canonical/vanilla-framework/assets/58276363/9eab36ac-f371-4eb1-897c-dd6eae154837)


Desktop
![image](https://github.com/canonical/vanilla-framework/assets/58276363/c7dea506-4b89-4370-9b07-8594f47ba99f)
